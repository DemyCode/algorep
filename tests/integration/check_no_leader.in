0
2
no
get_state 1
sleep 350
get_state 2
sleep 350
stop 1
stop 2