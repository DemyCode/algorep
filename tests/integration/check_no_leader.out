State 1: CANDIDATE
State 2: CANDIDATE